tmpl/tmpl.go

```diff
@@ -87,6 +89,12 @@ func Parse(s string, t time.Time) string {
 	hour := fmt.Sprintf("%02d", t.Hour())
 	s = regHour.ReplaceAllString(s, hour)
 
+	// {HOST}
+	if h, err := os.Hostname(); err == nil {
+		re := regexp.MustCompile(`(?i){host}`)
```

Comment:
regexp.MustCompile is super slow either set the regex as a var 
``` go
var hostRegex := regexp.MustCompile(`(?i){host}`)
```
or just do two strings.Replace 
``` go 
s = strings.Replace(s, "{host}")
s = strings.Replace(s, "{HOST}")
```

Reply:
I'm not sure I understand how `var hostRegex := regexp.MustCompile(`(?i){host}`)` is meaningfully different from `re := regexp.MustCompile(`(?i){host}`)`